src/CartItem.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import App from './App.js';
+import InventoryProd from './InventoryProd.js';
+
+class CartItem extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            //props: props
```

Comment:
Not sure if the comments in line 9 of `props` indicate that you plan on setting props as part of the initial state of `CartItem`? If so, uou may run into problems - this is generally considered an anti-pattern in React since it will not update state when/if the prop changes... introducing bugs... and also making it so there isn't a 'single source of truth' for this data in the app.